### PR TITLE
Remove Application Registry and Connector Service from FIT

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Make Fast Integration Tests not depend on Application Registry and Connector Service.

**Related issue(s)**
- Part of #13718

### CODE REVIEW

File: tests/fast-integration/test/fixtures/commerce-mock/index.js [hunk 1]
```diff
@@ -689,60 +667,6 @@ async function ensureCommerceMockLocalTestFixture(mockNamespace,
       mockNamespace,
       targetNamespace,
     withCentralApplicationConnectivity ? prepareFunction('central-app-gateway') : prepareFunction());
-  await waitForPodWithLabel('app.kubernetes.io/name', 'connector-service', 'kyma-integration');
-  await retryPromise(() => connectMockLocal(mockHost, targetNamespace), 10, 30000);
-  await retryPromise(() => registerAllApis(mockHost), 10, 30000);
-
-  if (withCentralApplicationConnectivity) {
-    await waitForDeployment('central-application-gateway', 'kyma-system');
-    await waitForDeployment('central-application-connectivity-validator', 'kyma-system');
```
Comment (franpog859) [inline tests/fast-integration/test/fixtures/commerce-mock/index.js:698]: I think we should not remove these. They should be up and running there and we should wait for it

File: tests/fast-integration/test/1-commerce-mock.js [hunk 2]
```diff
@@ -39,24 +33,6 @@ function commerceMockTests(testNamespace) {
...
-      await sendLegacyEventAndCheckResponse();
-    });
-
-    it('order.created.v1 cloud event in structured mode should trigger the lastorder function', async function() {
-      await sendCloudEventStructuredModeAndCheckResponse();
-    });
-
-    it('order.created.v1 cloud event in binary mode should trigger the lastorder function', async function() {
-      await sendCloudEventBinaryModeAndCheckResponse();
-    });
```
Comment (franpog859) [inline tests/fast-integration/test/1-commerce-mock.js:52]: Hey, are we sure we want to remove these two? I'm just checking, I remember that there was a discussion about it

File: tests/fast-integration/test/fixtures/commerce-mock/index.js [hunk 3]
```diff
@@ -107,7 +105,8 @@ function prepareFunction(type = 'standard', appName = 'commerce') {
           .replace('%%APP_NAME%%', appName));
     default:
       return k8s.loadAllYaml(functionYaml.toString()
-          .replace('%%URL%%', 'findEnv("GATEWAY_URL") + "/site/orders/" + code'));
+          .replace('%%URL%%', '"http://central-application-gateway.kyma-system:8080' +
+                   '/commerce/sap-commerce-cloud-commerce-webservices/site/orders/" + code'));
```
Comment (franpog859) [inline tests/fast-integration/test/fixtures/commerce-mock/index.js:109]: Hey, there is no difference between this one and the `central-app-gateway` case. Maybe we can just make it the default one? 
I assume there is no longer an option to use the old Application Gateway, right?

File: tests/fast-integration/test/fixtures/commerce-mock/application.yaml [hunk 4]
```diff
@@ -3,7 +3,53 @@ kind: Application
...
+        secretName: commerce-468ed067-dacb-4009-a9d5-497e3d2333c8
+        type: OAuth
+      targetUrl: http://commerce-mock.mocks.svc.cluster.local:10000/rest/v2
+      type: API
+    id: 468ed067-dacb-4009-a9d5-497e3d2333c8
+    labels:
+      connected-app: commerce
+    longDescription: These services manage all of the common commerce functionality,
+      and also include customizations from installed AddOns. The implementing extension
+      is called ycommercewebservices.
```
Comment (franpog859) [inline tests/fast-integration/test/fixtures/commerce-mock/application.yaml:29]: Should we adjust the description somehow? Maybe it's no longer that wide spectrum of functionalities anymore

**Review by franpog859 — COMMENTED** [review]
Please, ask also the owners of the tracing related components for a review as we remove their test there

File: tests/fast-integration/test/fixtures/commerce-mock/index.js (hunk 1 above)
Comment (VOID404) [reply tests/fast-integration/test/fixtures/commerce-mock/index.js:698]: We can wait for `central-application-gateway`, `connectivity-validator` stays only in compass scenario so we cannot really wait for it here

File: tests/fast-integration/test/fixtures/commerce-mock/index.js (hunk 3 above)
Comment (VOID404) [reply tests/fast-integration/test/fixtures/commerce-mock/index.js:109]: Or just make the `central-app-gateway` the default case, good point

File: tests/fast-integration/test/1-commerce-mock.js (hunk 2 above)
Comment (VOID404) [reply tests/fast-integration/test/1-commerce-mock.js:52]: Yes, cloud eventing doesn't work in open source scenario in current Kyma

**Review by franpog859 — APPROVED** [review]

**Review by pbochynski — APPROVED** [review]

**Comment by pbochynski** [comment]
Too old to be merged.

**Pull request closed without merging**